### PR TITLE
6087 removed phone mask and UX fixes

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#6087

*Description of changes:*
   - UX fixes of /bcgov/entity#6179

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/src/components/common/applicant-info-3.vue [hunk 1]
```diff
@@ -7,7 +7,6 @@
           <v-text-field :messages="messages['contact']"
                         :value="applicant.contact"
                         @blur="messages = {}"
-                        @focus="messages['contact'] = 'Contact Name (Optional)'"
```
Comment (severinbeauvais) [inline client/src/components/common/applicant-info-3.vue:10]: Out of curiousity, what did this do?

**Review by severinbeauvais — APPROVED** [review]

File: client/src/components/common/applicant-info-3.vue (hunk 1 above)
Comment (vysakh-menon-aot) [reply client/src/components/common/applicant-info-3.vue:10]: it show up in the position of validation error message and it is showing as red even after. ux issue raised by tracey
image

Comment (severinbeauvais) [reply client/src/components/common/applicant-info-3.vue:10]: Yeah, that was gross. I'm glad it's gone now :)

**Review by cameron-freshworks — APPROVED** [review]
Gg2!

**Pull request merged**